client/ayon_houdini/plugins/publish/collect_local_render_instances.py

```diff
@@ -134,17 +134,23 @@ def process(self, instance):
                                                context,
                                                colorspace=colorspace)
 
+            families = ["render.local.hou"]
+            if instance.data.get('review'):
+                families.append("review")
```

Comment:
I'm not sure what do these lines fix ? 

For information: 
`CollectLocalRenderInstances` was made to mimic what happens on the farm. 
if you submitted any render to farm via AYON, check the created json file, you will find that all instances have family `review` but not all of them have the tag `review`. and plugins should be able to filter based on the family and the tag.

Therefore, I'd say let's revert this part.

Reply:
Feel free to take over and push into the branch. As maintainer of the repo you should be able to push into the branch of the PR.

![image](https://github.com/user-attachments/assets/fe291ae5-c9c5-4861-87de-3673a40d90a8)

Reply:
I got an error about failed to push. 
and got this hint: 
```
hint: Updates were rejected because a pushed branch tip is behind its remote
```

and I'll avoid any push/pull commands because this PR is from a develop branch!

Reply:
@timsergeeff can you report what issue you faced with the `review` family still being present for an instance that had `instance.data["review"]` disabled?

Reply:
According to my testings: wo this code change all of files produced by (in my case) redshift rop will be getting review tag nomatter do i swith review toggle or not because when we create new instance per render product from reddhift_rop family we always tag it as review so even if i have several rops and only one with review turn on we get all products from all rops to have revirew and in my case i have a ton of posts in kitsu from all redhist rops

Reply:
> and I'll avoid any push/pull commands because this PR is from a develop branch!

sorry git is pretty new for me how can i change branch and what exactley is a problem? (just for my future understanding)

Reply:
> According to my testings: wo this code change all of files produced by (in my case) redshift rop will be getting review tag nomatter do i swith review toggle or not because when we create new instance per render product from reddhift_rop family we always tag it as review so even if i have several rops and only one with review turn on we get all products from all rops to have revirew and in my case i have a ton of posts in kitsu from all redhist rops

@MustafaJafar can you test this scenario and figure out _what_ is causing that instead? Is it that Kitsu logic may be disregarding whether representations have a 'review' tag? or?

Also, to me personally the code that @timsergeeff makes sense - why would it need `review` family if `instance.data["review"] = False` is what I wonder.

Reply:
> According to my testings: wo this code change all of files produced by (in my case) redshift rop will be getting review tag nomatter do i swith review toggle or not because when we create new instance per render product from reddhift_rop family we always tag it as review so even if i have several rops and only one with review turn on we get all products from all rops to have revirew and in my case i have a ton of posts in kitsu from all redhist rops

As I've mentioned earlier.
I'm only matching the results of the farm scripts, and please check the provided json file. and I believe it deserves its own PR because this PR was made to solve a particular issue.

FYI, we have `review` family and `review` tag.. 
I don't know why farm publishing add `review` family to all of the AOVs. but that's totally a different issue in my opinion.

Reply:
> > and I'll avoid any push/pull commands because this PR is from a develop branch!
> 
> sorry git is pretty new for me how can i change branch and what exactley is a problem? (just for my future understanding)

you need to create a new branch. please refer to PR name recommendations https://community.ynput.io/t/i-m-new-to-ayon-where-to-start/1070#how-to-collaborate-as-a-developer-14

Reply:
for reference, it should actually be as the suggestion below, because `instance.data.get('review')` will still add the review family for all the AOV instances. unlike the `preview` variable should only pick the AOVs that follows the regex pattern specified in the settings. 

```suggestion
            families = ["render.local.hou"]
            if preview:
                families.append("review")
```